tests/models/test_genericconfig.py

```diff
@@ -1,19 +1,46 @@
 """Tests for `oteapi.models.genericconfig`"""
-# pylint: disable=no-member
+# pylint: disable=no-member,pointless-statement,disallowed-name
```

Comment:
Isn't it better to define pylint disable where they matter rather than as a blanket statement at the top of the file.
I think this way we are teaching ourselves to just disregard pylint when it is inconvenient. I also think this way of doin it increases the risk of adding things that should not be disregarded in a future update.

Reply:
Ah, I missed that this was in the tests, where the main parts of pylint issues don't really apply. However, I can still move this down to be local if you wish?

Reply:
Yes, I still think it should be moved down, so that we do not unintentionally suggest bad practices (to e.g. me when I copy a test to make a new one).

Reply:
Moved it to the local places now :)